tests/install_vm.py

```diff
@@ -24,8 +24,8 @@ def parse_args():
     parser.add_argument(
         "--distro",
         dest="distro",
-        default="fedora",
-        help="What distribution to install (URL alias for fedora/centos7)"
+        required=True,
```

Comment:
please use choices here: https://docs.python.org/3/library/argparse.html#choices

Reply:
Good suggestion, thanks. I have added commit with choices for the `--distro`.